Content.Client/Xenoarchaeology/Ui/AnalysisConsoleMenu.xaml.cs

```diff
@@ -36,8 +37,14 @@ public sealed partial class AnalysisConsoleMenu : FancyWindow
     private Entity<XenoArtifactNodeComponent>? _currentNode;
 
     private TimeSpan? _hideExtractInfoIn;
-    private int _extractionSum;
 
+    // DeltaV
```

Comment:
Block comments should take the form `// Begin DeltaV - Explaination` and `// End DeltaV`